.github/workflows/ci.yml

```
@@ -43,7 +43,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Lint Commit Messages
-        uses: wagoid/commitlint-github-action@v3
+        uses: wagoid/commitlint-github-action@v5.3.1
```

```suggestion
        uses: wagoid/commitlint-github-action@v5
```

It seems we're supposed to use major version only and the tags get updated in the repository